PatchKit/Classes/Views/ImagesStackView.swift

```diff
@@ -0,0 +1,109 @@
+//
+//  ImagesStackView.swift
+//  abseil
+//
+//  Created by Omar Rasheed on 5/24/20.
+//
+
+import UIKit
+
+class ImagesStackView: UIView {
+    
+    private var imagesStackView = UIStackView()
+    
+    private var onImageTap: ((UIImage) -> Void)?
+    
+    override init(frame: CGRect) {
+        super.init(frame: .zero)
+        
+        setupImagesStackView()
+
+        setupConstraints()
+    }
+    
+    private func getImageRowStackView() -> UIStackView {
+        let stackView = UIStackView()
+        stackView.axis = .horizontal
+        stackView.distribution = .equalSpacing
+        stackView.spacing = 10.5
+        stackView.alignment = .center
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        
+        return stackView
+    }
+    
+    private func getImageView(for imageUrl: String, imageSize: CGSize) -> UIImageView {
+        let imageView = UIImageView()
+        imageView.backgroundColor = ._mediumGray
+        imageView.layer.cornerRadius = 6
+        imageView.clipsToBounds = true
+        imageView.fetchImage(imageUrl: imageUrl)
+        imageView.isUserInteractionEnabled = true
+        let tapGestureRecognizer = UITapGestureRecognizer(target: self, action: #selector(imageTapped(sender:)))
+        imageView.addGestureRecognizer(tapGestureRecognizer)
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+
+        NSLayoutConstraint.activate([
+            imageView.heightAnchor.constraint(equalToConstant: imageSize.height),
+            imageView.widthAnchor.constraint(equalToConstant: imageSize.width)
+        ])
+        
+        return imageView
+    }
+    
+    private func setupImagesStackView() {
+        imagesStackView.axis = .vertical
+        imagesStackView.distribution = .fillProportionally
+        imagesStackView.alignment = .leading
+        imagesStackView.spacing = 10
+        imagesStackView.translatesAutoresizingMaskIntoConstraints = false
+        addSubview(imagesStackView)
+    }
+    
+    func configure(for imageUrls: [String], imageSize: CGSize, onImageTap: ((UIImage) -> Void)? = nil) {
+        self.onImageTap = onImageTap
+        let numRows = Int(ceil(Double(imageUrls.count) / 3))
+        let numImagesPerRow = 3
```

Comment:
uhh im down to make it a param but i kinda didn't wanna mess w the math of like calculating max per row given size (if the two numbers would cause it to overflow) but i guess this problem already exists if we have size as a param. Ill change and just leave a TODO lol